deploy/dev-gcp.yml

```diff
@@ -290,29 +266,3 @@ spec:
 
     - name: PIP_USERS
       value: "srvPDP,srvk9-tilbake,srvsaf,srvk9-abakus,Z993210,Z994080,Z994692"
-
-    # Gruppenavn for roller fra Active Directory
```

Comment:
Brukes de ikke for å avgjøre hva innlogget bruker har lov til å gjøre, e.g. saksbehandle, veilede osv.?

Reply:
Never mind. Ser det er erstattet her: https://github.com/navikt/ung-sak/pull/204/files#diff-cc3197bcd34be824b3b5307a1df05165e75ad8c8d4b5e1e49e6ed2603cbacbaeR43